plugins/modules/fabric_devices_workflow_manager.py

```diff
@@ -0,0 +1,4080 @@
+                "ip_l3_handoff_details": "IP L3 Handoffs deleted successfully."
+            })
+        else:
+            result_fabric_device_msg.update({
+                "ip_l3_handoff_details": "IP L3 Handoffs are not found in the Cisco Catalyst Center."
+            })
+
+        self.msg = "Successfully deleted the IP L3 Handoffs."
+        self.status = "success"
+        return self
+
+    def delete_fabric_devices(self, fabric_devices):
+        """
+        Delete fabric devices, Border Settings, L2 Handoffs, L3 Handoff with SDA Transit,
+        L3 Handoff with IP Transit in Cisco Catalyst Center with fields provided in playbook.
+
+        Parameters:
+            fabric_devices (dict): SDA fabric devices under a fabric site playbook details.
+        Returns:
+            self (object): The current object with updated desired Fabric Devices information.
+        Description:
+
+        """
+
+        fabric_name = fabric_devices.get("fabric_name")
+        fabric_device_index = -1
+        device_config = fabric_devices.get("device_config")
+        self.response.append({"response": {}, "msg": {}})
+        self.response[0].get("response").update({fabric_name: {}})
+        self.response[0].get("msg").update({fabric_name: {}})
+        for item in device_config:
+            fabric_device_index += 1
+            device_ip = item.get("device_ip")
+            self.response[0].get("response").get(fabric_name).update({
+                device_ip: {
+                    "device_details": None,
+                    "l2_handoff": [],
+                    "l3_sda_handoff": None,
+                    "l3_ip_handoff": []
+                }
+            })
+            self.response[0].get("msg").get(fabric_name).update({
+                device_ip: {}
+            })
+            result_fabric_device_response = self.response[0].get("response").get(fabric_name).get(device_ip)
+            result_fabric_device_msg = self.response[0].get("msg").get(fabric_name).get(device_ip)
+            have_fabric_device = self.have.get("fabric_devices")[fabric_device_index]
+            have_l2_handoff = have_fabric_device.get("l2_handoff_details")
+            if item.get("l2_handoff_details"):
+                self.delete_l2_handoff(have_l2_handoff, device_ip,
+                                       result_fabric_device_response,
+                                       result_fabric_device_msg).check_return_status()
+
+            have_sda_l3_handoff = have_fabric_device.get("sda_l3_handoff_details")
+            if item.get("sda_l3_handoff_details"):
+                self.delete_sda_l3_handoff(have_sda_l3_handoff, device_ip,
+                                           result_fabric_device_response,
+                                           result_fabric_device_msg).check_return_status()
+
+            have_ip_l3_handoff = have_fabric_device.get("ip_l3_handoff_details")
+            if item.get("ip_l3_handoff_details"):
```

Comment:
self.log("Deleting IP L3 Handoff for device '{device_ip}'".format(device_ip=device_ip), "DEBUG")